roxctl/common/flags/endpoint.go

```diff
@@ -79,6 +81,9 @@ func AddConnectionFlags(c *cobra.Command) {
 func EndpointAndPlaintextSetting() (string, bool, error) {
```

Comment:
Let's state our intent clearly here:
* require port if scheme is not provided
* guess port from the scheme if provided

Reply:
fixed